junit_xml/__init__.py

```diff
@@ -317,6 +346,10 @@ def __init__(self, name, assertions=None, elapsed_sec=None,
         self.skipped_message = None
         self.skipped_output = None
 
+    def disable(self):
```

Comment:
It is not java, so it is not necessary to add setters/getters. If in the future you'd need  to add some logic you can easily do that converting `is_enabled` to a property. So I would drop that if that's was your concern